src/jit/compiler.h

```diff
@@ -281,17 +281,17 @@ class  LclVarDsc
     unsigned char       lvCustomLayout      :1;  // True when this struct has "CustomLayout"
     unsigned char       lvIsMultiRegArgOrRet:1; // Is this a struct that would be passed or returned in multiple registers?
 
-#ifdef _TARGET_ARM_
-    // TODO-Cleanup: Can this be subsumed by the above?
-    unsigned char       lvIsHfaRegArg:1;        // Is this argument variable holding a HFA register argument.
-    unsigned char       lvHfaTypeIsFloat:1;     // Is the HFA type float or double?
-#endif // _TARGET_ARM_
+#ifdef FEATURE_HFA
```

Comment:
If possible and not a lot of work, i would suggest to separate the the changes to replace _TARGET_ARM with FEATURE_HFA and associated refactoring changes into a PR of its own instead of clubbing with Arm64 work.